app/routes/_gcn.docs.contributing._index.mdx

```diff
@@ -60,6 +60,18 @@ Here are instructions for getting the GCN site set up and running on your own co
     -   Install and use the [Node Version Manager, nvm](https://github.com/nvm-sh/nvm).
     -   Install using a [package manager](https://nodejs.org/en/download/package-manager/).
 
+  </ProcessListItem>
+  <ProcessListItem>
+    <ProcessListHeading type="h3">
+      Set up Python on your computer
+    </ProcessListHeading>
+
+    Make sure that Python 3.9 or newer is installed on your computer. There are several Python distributions available depending on your operating system. Pick _one_ of the following:
+
+    -   If you are new to Python, then we suggest the [Anaconda Distribution](https://www.anaconda.com/download).
+    -   For other distributions and installation methods, see the [Python Setup and Usage](https://docs.python.org/3/using/index.html) guide in the official Python documentation.
+    -   If you have a sufficiently new version of Linux or macOS, then a suitable version of Python may come pre-installed on your system. To check, run `python3 --version`.
```

Comment:
Nit: Not sure how specific we should be here since @jak574 pointed out from my error message that it was `pip` which was missing, but I presume any updates to python will also fix this.

Reply:
@Kirill-Vorobyev pip is included by default in every user installation of python (>3.4 I think), so just installing python>3.9 will satisfy the error message you received.